resource/macos/scheme-patch.diff

```diff
@@ -0,0 +1,14 @@
+--- ./node_modules/app-builder-lib/scheme.json	1985-10-26 09:15:00.000000000 +0100
++++ /Users/lore/Desktop/scheme.json	2019-10-14 12:54:44.000000000 +0200
```

Comment:
Why this change? This will only work on your system.

Reply:
Damn! Good catch! Thank you!
It doesn't actually affect the build because we are applying the patches so it only uses the first line, but it's pretty bad to have. 
I've fixed it.